File: packages/js/node/core-rs/test/auth-cli.ts
import { canonicalScopes, credentialKey } from "../index.ts";

const scopes = canonicalScopes(process.argv.slice(2));
console.log(
  JSON.stringify({
    provider: "example",
    profile: null,
    scopes,
    credentialKey: credentialKey("example", undefined, scopes),
  }),
);

File: packages/py/core-rs/test/auth_cli.py
import json
import sys

from dbx_tools.core_rs.bindings import canonical_scopes, credential_key

scopes = canonical_scopes(sys.argv[1:])
print(
    json.dumps(
        {
            "provider": "example",
            "profile": None,
            "scopes": scopes,
            "credentialKey": credential_key("example", None, scopes),
        }
    )
)
